docroot/modules/custom/civictheme_migrate/src/Form/MigrationEditFormAlter.php

```diff
@@ -84,7 +84,7 @@ public function formAlter(array &$form, FormStateInterface &$form_state, string
     ];
 
     $entity_type_id = str_replace('entity:', '', $migration_entity->destination['plugin'] ?? '');
-    $bundle = $migration_entity->destination['default_bundle'] ?? '';
+    $bundle = $migration_entity->destination['default_bundle'] ?? $migration_entity->destination['validation_bundle'] ?? '';
```

Comment:
Cannot  update 
EntityContentBase $entity_type = static::getEntityTypeId($plugin_id);

Though of this as an alternative way to do it.